service/src/java/org/apache/hive/service/cli/HiveSQLException.java

```diff
@@ -58,6 +60,10 @@ public HiveSQLException(String reason) {
     super(reason);
   }
 
+  public HiveSQLException(String reason, QueryState queryState) {
+    super(ServiceUtils.addQueryId(reason, queryState), reason);
```

Comment:
i think ServiceUtils should provide `getQueryId` method and final message could be constructed in HiveSQLException

Reply:
Done